### ISSUE TITLE
Motionless  targets has speed and course

### ISSUE THREAD

**Issue by Hakansv** [douwefokkema/marpa#8]
This is probably a simulation effect but noted for further real tests on board.
If still an issue after live tests please consider a more efficient filter to detect motionless targets and force speed and course to zero.

**Comment by douwefokkema** [comment]
Can you give some examples of the values you get for speed and course for a stationary target?

**Comment by Hakansv** [comment]
Well, since we are simulating the boat's speed is not as it was during capture so we're moving relative the target in a false way. Set a target on a buoy and see yourself. But here is an example:
Maybe we would send a speed below 0.5? knots as zero and just update range and bearing?

![fixarpa](https://cloud.githubusercontent.com/assets/7202854/20631894/9188d8d8-b339-11e6-8975-9239455404f5.PNG)
.

**Comment by douwefokkema** [comment]
You are completely right, I also get speeds of up to .3 kn on a steady target. I will first work on improvement of the filter. I needed we can later set low speeds to 0.

**Comment by Hakansv** [comment]
More observation, now on board my moored boat.
Fixed targets like a rock is still moving and get speed and course. Maybe not less due to my boat is "jumping" around a little. That's due to a varying GPS position I suppose. But as of the pictures below the range and bearing differs a little over time and of course that's evaluated to a movement.
Targets speed was varying between 1.06 knots and 0.15. The latter showed no prediction arrow.
Course and speed is fluctuating a lot. The arrow can rotate a full circle in one minute but also no arrow at rare occasions.
How to adapt a filter for this? Discussion:
Maybe if the average speed has been below one knot the last XX measurements AND the course alternates more than XX(90?) degrees we should set the reported speed to zero? OR in general if the speed is below 0.45?
Two shoots, not more than 30 seconds in between.
![fixtarget2](https://cloud.githubusercontent.com/assets/7202854/20649278/1244097e-b4bc-11e6-93af-5d005b603968.PNG)

**Comment by douwefokkema** [comment]
Solution for this issue implemented as follows:
There is now a #define TARGET_SPEED_DIV_SDEV 2.  // when speed is < TARGET_SPEED_DIV_SDEV * standard_deviation of speed, speed of target  is shown as 0
So the speed is compared with its standard deviation. If smaller than the factor defined above, speed and course are considered to be unreliable and are set to 0 for display in the OCPN target query.
Needs additional test
